src/themes/teams/components/List/listItemVariables.ts

```diff
@@ -1,6 +1,7 @@
 export default siteVariables => ({
   // Header
   // TODO: prod app uses 17.5px here, it should be 16px per the design guide!
+  headerColor: siteVariables.gray02,
```

Comment:
Can you move the comment, so it is above the line that is actually about.

Reply:
oups :D